pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -140,6 +140,13 @@ buildDotnetModule (finalAttrs: {
 
       # Fails with: Expected a <System.ArgumentException> to be thrown, but no exception was thrown.
       "NexusMods.Networking.ModUpdates.Tests.PerFeedCacheUpdaterTests.Constructor_WithItemsFromDifferentGames_ShouldThrowArgumentException_InDebug"
+
+      # Require networking but aren't marked as RequiresNetworking=True
+      "NexusMods.DataModel.SchemaVersions.Tests.LegacyDatabaseSupportTests.TestDatabase"
+      "NexusMods.DataModel.SchemaVersions.Tests.MigrationSpecificTests.TestsFor_0001_ConvertTimestamps.OldTimestampsAreInRange"
+      "NexusMods.DataModel.SchemaVersions.Tests.MigrationSpecificTests.TestsFor_0003_FixDuplicates.No_Duplicates"
+      "NexusMods.DataModel.SchemaVersions.Tests.MigrationSpecificTests.TestsFor_0004_RemoveGameFiles.Test"
+      "NexusMods.Games.StardewValley.Tests.SMAPIGameVersionDiagnosticEmitterTests.Test_TryGetLastSupportedSMAPIVersion"
```

Comment:
Ah, I see. According to https://github.com/Nexus-Mods/NexusMods.App/pull/1222#issuecomment-2060687094, `RequiresNetworking` is just poorly named. The trait is actually only supposed to apply to tests that require the Nexus Mods API, not networking more generally.